datalad/core/local/tests/test_save.py

```diff
@@ -593,6 +594,25 @@ def test_partial_unlocked(path):
     assert_repo_status(ds.path)
 
 
+@known_failure
```

Comment:
I prefer to label this as a known failure, but I suspect others might prefer that we not skip the test and instead assert that the rev_save call fails with a CommandError.  I'm ok making that change, and I'm also fine with dropping this PR entirely if others don't think this test is worth carrying.

Reply:
On second thought, using `@known_failure` isn't worth it since there isn't even a known failure probe that runs on master.  I've updated it to assert that a CommandError is raised.  (I've also clarified some bits of the commit message.)